diff --git a/node_modules/react-native-audio-api/common/cpp/audioapi/core/utils/Constants.h b/node_modules/react-native-audio-api/common/cpp/audioapi/core/utils/Constants.h
index 2cf3665..b7e0f3a 100644
--- a/node_modules/react-native-audio-api/common/cpp/audioapi/core/utils/Constants.h
+++ b/node_modules/react-native-audio-api/common/cpp/audioapi/core/utils/Constants.h
@@ -42,5 +42,5 @@ constexpr std::size_t hardware_destructive_interference_size = 64;
 #endif
 
 // audio param
-inline constexpr size_t AUDIO_PARAM_MAX_QUEUED_EVENTS = 64;
+inline constexpr size_t AUDIO_PARAM_MAX_QUEUED_EVENTS = 512;
 } // namespace audioapi
